feature/security/gnsi/certz/tests/client_certificates/client_certificates_test.go

```diff
@@ -0,0 +1,254 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client_certificates_test
+
+import (
+	context "context"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+	"os"
+	"testing"
+
+	setupService "github.com/openconfig/featureprofiles/feature/security/gnsi/certz/tests/internal/setup_service"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygot/ygot"
+)
+
+var (
+	testProfile = "gNxI"
```

Comment:
@priyacj the spec was updated earlier today to indicate that the default profile should not be modified
https://github.com/openconfig/gnsi/blob/80891f288c18591a5930b4acd8ad5ae2c9504472/certz/README.md?plain=1#L96-L102; could you please update the test accordingly?
thx,

Reply:
What's "the next upcoming merge"? Are you suggesting that it shouldn't be a part of this PR (if so, why)?

Reply:
The default gNxI profile is been replaced with new sslprofile <<newprofile>>. This ssl profile is added using addSSLprofile and then used for rotate RPC.